templates/scenarios/js/dashboard-tab/package.json.tpl

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "{%appName%}",
+    "version": "0.1.0",
+    "engines": {
+        "node": ">=14 <=16"
+    },
+    "private": true,
+    "dependencies": {
+       "@fluentui/react": "^8.99.2",
+        "@fluentui/react-charting": "^5.14.10",
+        "@fluentui/react-components": "^9.7.1",
+        "@fluentui/react-icons": "^2.0.186",
+        "@fluentui/react-northstar": "0.62.0",
+        "@fluentui/react-theme": "^9.1.3",
+        "@microsoft/microsoft-graph-client": "^3.0.1",
+        "@microsoft/teams-js": "^2.2.0",
+        "@microsoft/teamsfx": "^2.0.1",
+        "@microsoft/teamsfx-react": "^2.0.1",
+        "@react-hook/resize-observer": "^1.2.6",
+        "axios": "^0.21.1",
+        "msteams-react-base-component": "^4.0.1",
+        "react": "^16.14.0",
+        "react-dom": "^16.14.0",
+        "react-router-dom": "^5.1.2",
+        "react-scripts": "^5.0.1"
+    },
+    "devDependencies": {
```

Comment:
@types/ dependencies are not needed since it's a js project.

Reply:
Thanks for helping to point out this problem, I have fixed it.